public/kubernetes-guides/advanced-guides/dynamic-resource-allocation.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Dynamic Resource Allocation"
+description: "Request and share node level resources among Kubernetes pods."
+---
+
+[Kubernetes Dynamic Resource Allocation](https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA) is a new way of sharing host level resources into pods.
+These resources include GPU devices, high performance networking, and other hardware that a workload may need access to.
+
+DRA is enabled by default as a beta feature in Kubernetes 1.34 and can also be enabled in Kubernetes 1.33.
+DRA replaces [devices plugins](./device-plugins.mdx) for accesing resources from workloads.
+
+## Enable DRA in the cluster
+
+Make sure the cluster has DRA enabled (Kubernetes 1.34+) or uses the following patch to enable the feature. This patch can be applied to all nodes in the cluster via `talosctl` or via Omni.
+
+```yaml
+machine:
+  kubelet:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+cluster:
+  apiServer:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+  controllerManager:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+  scheduler:
+    extraArgs:
+      feature-gates: DynamicResourceAllocation=true
+```
+
+You should have at least one node in the cluster with NVIDIA hardware and configured via the [NVIDIA system extension and patch](../../talos/v1.11/configure-your-talos-cluster/hardware-and-drivers/nvidia-gpu-proprietary).
+
+## Deploy the NVIDIA DRA plugin
+
+### Use `helm` to deploy the DRA driver:
+
+```sh
+helm repo add nvidia https://helm.ngc.nvidia.com/nvidia
+helm repo update
+```
+
+Install the driver in the kube-system namespace or create a new namespace and label it with appropriate [Pod Security Admission](../security/pod-security.mdx).
+
+```sh
+helm install nvidia-dra-driver-gpu nvidia/nvidia-dra-driver-gpu \
+    --version="25.8.0" \
+    --namespace kube-system \
+    --set resources.gpus.enabled=false \
+    --set nvidiaDriverRoot=/usr/local/glibc/lib
+```
+
+Once the kubelet plugin pod is running you can verify your resourceslices objects.
+
+```sh
+kubectl get pods -n kube-system -l app.kubernetes.io/name=nvidia-dra-driver-gpu
+```
+
+### Verify ResourceSlice objects
```

Comment:
wouldnt this headers be better as steps. instead of their own sub-sections. They seem a lot more sequential. Like do this, then do this.